### ISSUE TITLE
error: Missing type parameters for generic type "File"

### ISSUE THREAD

**Issue by mazlum** [typeddjango/django-stubs#1061]
# Bug report

## What's wrong
```python
from django.core.files import File

def __call__(self, value: File) -> None:
```

Mypy says:
Missing type parameters for generic type "File"

If I use like that:
```python
def __call__(self, value: File[AnyStr]) -> None:
```
Result is:
TypeError: 'type' object is not subscriptable

**Comment by ngnpope** [comment]
This will be fixed in the next release of `django-stubs-ext`, see https://github.com/typeddjango/django-stubs/commit/e8d52fe7daf88cfe7db5d0dd312e1b5946577051. In the meantime, you can use the `extra_classes` argument of `django_stubs_ext.monkeypatch()`.

**Comment by intgr** [comment]
ngnpope said this should be fixed by #1050. Can this be closed now?

**Comment by flaeppe** [comment]
https://github.com/typeddjango/django-stubs/issues/1061#issuecomment-1307163708
